packages/kbn-search-api-panels/components/cloud_details.tsx

```diff
@@ -97,7 +97,7 @@ export const CloudDetailsPanel = ({
             </EuiFlexItem>
             <EuiFlexItem grow={false}>
               <span>
-                <EuiBadge color="success">
+                <EuiBadge color="accentSecondary">
```

Comment:
As far as I've been told, latest design decision is that EuiBadge should not support `accentSecondary`, we're removing it in the next Borealis version ([EUI PR](https://github.com/elastic/eui/pull/8227)).
The guidance is to keep `success` in place.

cc @ek-so